content/en/docs/releases/supported-releases/index.md

```diff
@@ -49,6 +50,12 @@ Our naming scheme is as follows:
 where `<minor>` is increased for each release, and `<patch>` counts the number of patches for the
 current `<minor>` release. A patch is usually a small change relative to the `<minor>` release.
 
+## Control Plane/Data Plane Skew
+
+The Istio control plane can be 1 version ahead of the data plane. However, the data plane cannot be ahead of control plane. We recommend using [revisions](/docs/setup/upgrade/canary/) so that there is no skew at all.
```

Comment:
```suggestion
The Istio control plane can be one version ahead of the data plane. However, the data plane cannot be ahead of control plane. We recommend using [revisions](/docs/setup/upgrade/canary/) so that there is no skew at all.
```